packages/taquito/test/contract/rpc-contract-provider.spec.ts

```diff
@@ -2323,4 +2328,31 @@ describe('RpcContractProvider test', () => {
       done();
     });
   });
+
+  describe('smartRollupOriginate', () => {
```

Comment:
please follow the existing test patterns of checking the `opOb` being outputted, and keep in mind the 3 cases:
- op with reveal
- op without reveal
- op with fees (or other estimates) overridden

Reply:
hmmm i definitely added this but dont see it 👀 i'll add it again

Reply:
not sure what happened to my commit that fixed this 😦